server/services/exhibitions.js

```diff
@@ -48,24 +57,27 @@ export async function getExhibition(id: string, previewReq: ?Request): Promise<?
   const sizeInKb = Math.round(exhibition.data.textAndCaptionsDocument.size / 1024);
   const textAndCaptionsDocument = Object.assign({}, exhibition.data.textAndCaptionsDocument, {sizeInKb});
 
-  return ({
-    blockType: 'exhibitions',
+  const promo = getPromo(exhibition);
+
+  const ex = ({
     id: exhibition.id,
-    title: RichText.asText(exhibition.data.title),
+    title: asText(exhibition.data.title),
+    subtitle: asText(exhibition.data.subtitle),
     start: exhibition.data.start,
     end: exhibition.data.end,
-    featuredImages: featuredImages,
-    accessStatements: exhibition.data.accessStatements,
-    description: exhibition.data.description && RichText.asHtml(exhibition.data.description),
-    video: video,
-    standfirst: standfirst,
-    text: text,
+    featuredImages: List(featuredImages),
+    description: asHtml(exhibition.data.description),
+    promo: promo
+  }: Exhibition);
+
+  return {
+    exhibition: ex,
     imageGallery: imageGallery,
-    galleryLevel: exhibition.data.gallery_level,
+    galleryLevel: '0',
```

Comment:
Going down the route of we haven't had this problem occur more than once so don't fully understand the requirements, thus making it configurable seems premature.

Once we understand the domain more, we can come up with a more fully-fledged solution (I imagine something like attaching a `GallerySpace` or something, but unsure).